.github/workflows/macos.yml

```diff
@@ -91,7 +91,10 @@ jobs:
         uses: ./.github/actions/launchable/setup
         with:
           launchable-token: ${{ secrets.LAUNCHABLE_TOKEN }}
+          os: ${{ matrix.os }}
+          test-opts: ${{ matrix.test_opts }}
           srcdir: src
+          builddir: build
```

Comment:
The order of the options is os, test-opts, launchable-token, builddir, srcdir. We can align the order here, also to align with the one in the `ubuntu.yml`.